Microsoft/WebTemplateStudio
# Add Privacy Modal to Licenses

Task: https://microsoftgarage.visualstudio.com/Intern%20GitHub/_workitems/edit/30921

**Changes**
- Adds a Privacy Modal to Licenses

**How to Check**
- Open the wizard.
- Select Frameworks and pages
- Open the licenses modal from the right side bar.
- Click on any license and it should show up the privacy modal for that license.
